src/browser/modules/D3Visualization/lib/visualization/components/Visualization.ts

```diff
@@ -20,8 +20,13 @@
 import { easeCubic } from 'd3-ease'
 import { BaseType, Selection, select as d3Select } from 'd3-selection'
 import { D3ZoomEvent, ZoomBehavior, zoom as d3Zoom } from 'd3-zoom'
+import * as d3Zooming from 'd3-zoom'
```

Comment:
Unsure if we're doing any tree shaking, but even just for consistency it would be best to import only the functions we're using.

```suggestion
import { D3ZoomEvent, ZoomBehavior, zoom as d3Zoom, zoomIdentity } from 'd3-zoom'
```